runelite-client/src/main/java/net/runelite/client/plugins/hpandprayeroverlay/hpAndPrayerOverlay.java

```diff
@@ -0,0 +1,160 @@
+    }
+
+    @Inject
+    private SkillIconManager iconManager;
+
+
+    @Override
+    public Dimension render(Graphics2D g) {
+
+
+        int realHpLevel = client.getRealSkillLevel(Skill.HITPOINTS);
+        int realPrayerLevel = client.getRealSkillLevel(Skill.PRAYER);
+        int boostedHpLevel = client.getBoostedSkillLevel(Skill.HITPOINTS);
+        int boostedPrayerLevel = client.getBoostedSkillLevel(Skill.PRAYER);
+        int healthBarheight = (boostedHpLevel * 224) / realHpLevel;
+        int prayerBarheight = (boostedPrayerLevel * 224) / realPrayerLevel;
+        int healthBarPosition = 455 - healthBarheight;
+        int prayerBarPosition = 453 - prayerBarheight;
+
+
+
+        if (config.enableHP()){
+
+            // Fixed bars for fixed client
+            g.setColor(Color.BLACK);
+            g.fillRect(530, 229,13,228);
+            g.setColor(Color.red);
+            g.fillRect(532, 231,9,224);
+            g.drawImage(iconManager.getSkillImage(Skill.HITPOINTS),527,209,20,18,null);
+
+            // dynamic bar
+            g.setColor(Color.GREEN);
+            g.fillRect(532, healthBarPosition,9, healthBarheight);
+        }
+
+        if (config.enablePray()){
+
+            // Fixed bars for fixed client
+            g.setColor(Color.BLACK);
+            g.fillRect(741, 227,13,228);
+            g.setColor(Color.BLUE);
+            g.fillRect(743, 229,9,224);
+            g.drawImage(iconManager.getSkillImage(Skill.PRAYER),736,204,23,23,null);
+
+            //dynamic bar
+            g.setColor(Color.CYAN);
+            g.fillRect(743, prayerBarPosition,9,prayerBarheight);
+
+        }
+
+        if (client.isResized()){
+            setPosition(OverlayPosition.BOTTOM_RIGHT);
+
+            Color c=new Color(90,91,94,127 );
+            g.setColor(c);
+            g.fillRect(-43, -133,53,258);
+
+            if (config.enableHP()){
+
+                // Fixed bars for fixed client
+                g.setColor(Color.BLACK);
+                g.fillRect(-34, -106,13,228);
+                g.setColor(Color.red);
+                g.fillRect(-32, -104,9,224);
+                g.drawImage(iconManager.getSkillImage(Skill.HITPOINTS),-37,-125,20,18,null);
+
+                // dynamic bar
+                g.setColor(Color.GREEN);
+                g.fillRect(-32, 120 - healthBarheight,9, healthBarheight);
+            }
+
+            if (config.enablePray()){
+                // Fixed bars for fixed client
+                g.setColor(Color.BLACK);
+                g.fillRect(-10, -106,13,228);
+                g.setColor(Color.BLUE);
+                g.fillRect(-8, -104,9,224);
+                g.drawImage(iconManager.getSkillImage(Skill.PRAYER),-15,-130,23,23,null);
+
+                //dynamic bar
+                g.setColor(Color.CYAN);
+                g.fillRect(0 - 8,  120 - prayerBarheight,9,prayerBarheight);
+            }
+        }
+        else
+            setPosition(OverlayPosition.DYNAMIC);

```

Comment:
You should use brackets on this else statement to remain consistent with the codebase.